src/monitor.rs

```diff
@@ -885,7 +885,10 @@ impl UdevMonitor {
                     events: libc::POLLIN,
                     revents: 0,
                 }];
+                #[cfg(target_pointer_width = "64")]
                 let pfd_len = pfd.len() as u64;
+                #[cfg(target_pointer_width = "32")]
+                let pfd_len = pfd.len() as u32;
```

Comment:
Could you please add a CI runner for 32-bit systems?

If not, I can add one.

Reply:
That's a good suggestion, but github actions are really not my strong suite.

A quick google seems to indicate that this is non-trivial: https://github.com/actions/runner/issues/1181

Reply:
Alright, well the link you provided ended up being a bit of a dead-end.

Luckily, we can just use a Rust target for `i686-unknown-linux-gnu` to test for 32-bit.

Please rebase your changes on https://github.com/cr8t/udev/pull/30.

Reply:
Once CI is added + passes for these changes, we can add `i586-unknown-linux-gnu` and `i686-unknown-linux-gnu` to supported platforms in #31.